ui-ngx/src/app/modules/home/pages/dashboard/dashboard-form.component.html

```diff
@@ -124,19 +124,28 @@
           <textarea matInput formControlName="description" rows="2"></textarea>
         </mat-form-field>
       </div>
-      <div translate>dashboard.mobile-app-settings</div>
-      <tb-image-input fxFlex
-                      label="{{'dashboard.image' | translate}}"
-                      maxSizeByte="524288"
-                      formControlName="image">
-      </tb-image-input>
-      <mat-checkbox fxFlex formControlName="mobileHide">
-        {{ 'dashboard.mobile-hide' | translate }}
-      </mat-checkbox>
-      <mat-form-field class="mat-block">
-        <mat-label translate>dashboard.mobile-order</mat-label>
-        <input matInput formControlName="mobileOrder" type="number" step="1">
-      </mat-form-field>
+      <div fxLayout="column" [fxShow]="isAdd">
+        <span>{{ 'dashboard.assigned-customers' | translate }}</span>
+        <tb-entity-list
+          formControlName="assignedCustomerIds"
+          [entityType]="entityType.CUSTOMER">
+        </tb-entity-list>
```

Comment:
This is a bad design. When using material styles, the field with the name cannot be located after the input element.